configs/components/nssm.rb

```diff
@@ -0,0 +1,12 @@
+component "nssm" do |pkg, settings, platform|
+  pkg.load_from_json("configs/components/nssm.json")
+
+  # Because we're unpacking a zip archive, we need to set the path to the executable.
+  # We don't automatically have this set on windows, unfortunately. We need to set the
+  # path to have access to 7za.
+  pkg.environment "PATH" => "/cygdrive/c/ProgramData/chocolatey/tools:$$PATH" if platform.is_windows?
```

Comment:
Is this something we should be setting at the platform level?  Seems like we may need this in other components.